### PR TITLE
feat: sorting machine improvements based of user testing

### PR DESCRIPTION
**Pull request by Herman1412**
add a screen displaying the color of the tier, made the machine larger and added minor improvements, lowered roof

close: #452 and #454

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
feature

* **What is the current behavior?** (You can also link to an open issue here)
sorting machine has no indication of tier and fish gets stuck in it. See #452  and #454 

* **What is the new behavior?** (if this is a feature change)
indicator screen has been added and changes color based on what tier is active. all objects the fish touches is frictionless

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

### CODE REVIEW

File: Assets/FishFactory/Scripts/FishSortingButton.cs [hunk 1]
```diff
@@ -18,28 +18,48 @@ public enum FishTier
...
+        tier1 = Resources.Load<Material>("Materials/QAPannel/Tier1");
+        tier2 = Resources.Load<Material>("Materials/QAPannel/Tier2");
+        tier3 = Resources.Load<Material>("Materials/QAPannel/Tier3");
+
+        // Sets the instance of the FishSortingButton to this object if it does not already exist
+        if (fishSortingButton == null)
+        {
+            fishSortingButton = this;
+        }
+        // Find first instance of ColorBox, if there are more than one sorting machine that the player should be able to use this will need to be changed
```
Comment (jonp-h) [inline Assets/FishFactory/Scripts/FishSortingButton.cs:42]: What does this mean? I don't understand

File: Assets/FishFactory/Scripts/FishSortingButton.cs [hunk 2]
```diff
@@ -18,28 +18,48 @@ public enum FishTier
     // list of sorted fish
     public List<GameObject> sortedFish = new List<GameObject>();
 
+    // ------------------ Private Variables ------------------
+
+    private Material tier1;
+    private Material tier2;
+    private Material tier3;
+    private GameObject ColorPannel;
```
Comment (jonp-h) [inline Assets/FishFactory/Scripts/FishSortingButton.cs:26]: Panel*

**Review by jonp-h — COMMENTED** [review]
Read comments before merge

File: Assets/FishFactory/Scripts/FishSortingButton.cs (hunk 1 above)
Comment (Herman1412) [reply Assets/FishFactory/Scripts/FishSortingButton.cs:42]: .Find() gets only one game object, if there are more than 1 game object with the same name then the second one will not change color. This is preferable in our case but might be a problem if the user needs to interact with multiple different sorting machines.

**Review by jonp-h — APPROVED** [review]

**Pull request merged**